model/src/wav_shel_inp.F90

```diff
@@ -258,17 +258,30 @@ subroutine read_shel_config(mpi_comm, mds, time0_overwrite, timen_overwrite)
     ! Read nml file if available
     !--------------------
 
+#ifndef W3_CESMCOUPLED
     inquire(file=trim(fnmpre)//"ww3_shel.nml", exist=flgnml)
+#else
+    inquire(file=trim(fnmpre)//"wav_in", exist=flgnml)
+#endif
```

Comment:
It would be nice if the filename was resolved in one place (into a local variable) and then used here and below. That way, the code difference between models is minimized.
-- Committee for less ifdef code
```suggestion
#ifndef W3_CESMCOUPLED
    nl_filename = trim(fnmpre)//"ww3_shel.nml"
#else
    nl_filename = trim(fnmpre)//"wav_in"
#endif

    inquire(file=trim(nl_filename), exist=flgnml)
```

Reply:
@gold2718 - I agree. I had done that originally and had other problems associated with changing the namelist.
I'll implement your suggestion now.